save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPluginConfig.kt

```diff
@@ -51,3 +65,16 @@ data class FixPluginConfig(
         internal val defaultResourceNamePattern = Regex("""(.+)(Expected|Test)\.[\w\d]+""")
     }
 }
+
+@OptIn(ExperimentalSerializationApi::class)
+@Serializer(forClass = Regex::class)
+object RegexSerializer : KSerializer<Regex> {
```

Comment:
Let's move it to `save-common`, because it's now duplicated in two places already